enterprise/j2ee.earproject/src/org/netbeans/modules/j2ee/earproject/ProjectEar.java

```diff
@@ -885,7 +924,7 @@ public void artifactsUpdated(Iterable<Artifact> artifacts) {
                     try {
                         FileUtil.createData(destFile);
                     } catch (IOException ex) {
-                        LOGGER.log(Level.INFO, "Could not prepare data file", ex);
+                        LOGGER.log(Level.INFO, "Could not prepare data file {0}", ex);
```

Comment:
The parameter is not necessary. If an exception is present, it will be logged with message, no need to for the placeholder.

Reply:
Fixed it. Thanks.